tests/services/conftest.py

```diff
@@ -166,3 +166,25 @@ def analysis_without_jobs(analysis_store: Store):
 @pytest.fixture
 def analysis_service(analysis_store: Store, job_service_mock: JobService) -> AnalysisService:
     return AnalysisService(store=analysis_store, job_service=job_service_mock)
+
+
+@pytest.fixture
+def tower_analysis_without_jobs_and_pending(analysis_store: Store):
+    analysis = Analysis(
+        config_path="config_path",
+        workflow="workflow",
+        case_id="case_id",
+        out_dir="out_dir",
+        priority=PRIORITY_OPTIONS[0],
+        started_at=datetime.now() - timedelta(weeks=1),
+        status=TrailblazerStatus.PENDING,
+        ticket_id="ticket_id",
+        type=TYPES[0],
+        workflow_manager=WorkflowManager.TOWER,
+        is_visible=True,
+        order_id=1,
+    )
+    session: Session = get_session()
+    session.add(analysis)
+    session.commit()
```

Comment:
Check out the createhandler in trailblazer. It has a function add pending analysis. We would want to use that here to add the analysis to the store. Otherwise this fixture is now coupled to the implementation of the store and has crud layer functionality within it. Ideally we do not want that.